app/views/reset_password/new.html.haml

```diff
@@ -0,0 +1,21 @@
+#header
+  .container
+    .row
+      .col-lg-4.col-lg-offset-4.col-md-6.col-md-offset-3.col-sm-8.col-sm-offset-2
+        %h1
+          %a{:href => root_path()} Epic Logger
+        .panel.panel-default
+          %h3 Email
+          = simple_form_for :user, url: reset_password_index_url, method: :post, wrapper: :vertical_form do |f|
```

Comment:
this url is wrong. Why are you having an url that has index in it's name? What does index have to do with anything?